### PR TITLE
Adding Logs-To API,Lamda and Website

### PR DESCRIPTION
**Pull request by mayajohns**
### Requirements

filter the logs based on Day,month,year etc after the deployments of Api,Lamda and website

### Benefits

Able to see the details logs based on all the available scenarios.
### Possible Drawbacks

Through automation,we have set certain order,based on that only logs will visible

### CODE REVIEW

File: core/jazz_ui/e2e/page-objects/jazzservices.po.ts [hunk 1]
```diff
@@ -115,5 +116,26 @@ export class Jazz {
...
+    return element(by.css('section > env-logs-section > div > div > div.drp-wrp > div:nth-child(2) > div > div > div > div > span'));
+  }
+  getDay(){
+    return element(by.css('div.eachFilter.slider-wrap > dropdown > div > ul > li:nth-child(1) > a'));
+  }
+  getWeek(){
+    return element(by.css('div.eachFilter.slider-wrap > dropdown > div > ul > li:nth-child(2) > a'));
+  }
+  getMonth(){
+    return element(by.css('div.eachFilter.slider-wrap > dropdown > div > ul > li:nth-child(3) > a'));
```
Comment (svidhani) [inline core/jazz_ui/e2e/page-objects/jazzservices.po.ts]: ya this is nasty. We will have to do something about that in the source code

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts [hunk 2]
```diff
@@ -217,6 +257,26 @@ describe('Overview', () => {
...
+
+      it('Verify Logs' ,  () => {
+            browser.wait(EC.visibilityOf(jazzServices_po.getLogs()), timeOutHigh);
+            jazzServices_po.getLogs().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getFilterIcon()), timeOutHigh);
+            browser.driver.switchTo().activeElement();
+            jazzServices_po.getFilterIcon().click();
+            jazzServices_po.getDropDown().click();
+            jazzServices_po.getDay().click();
+            browser.sleep(4000);
```
Comment (svidhani) [inline core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: do you really need 4 seconds of sleep here and below

File: core/jazz_ui/e2e/page-objects/jazzservices.po.ts [hunk 3]
```diff
@@ -81,12 +81,12 @@ export class Jazz {
...
   getProdName(){
     return element(by.xpath('//div[@class="stage-title stageDisp" and contains(text(),"prod")]'))
-  }
+	}
   getProdHeader(){
     return element(by.xpath('//div[@class="servie-details-container"]/h1[@class="page-hdr bold-title relative" and contains(text(),"prod")]'));
   }
   getAsset(){
-    return element(by.xpath('//div/div[2]/div/section[1]/div/div[2]/tabs/div/ul/li[3]'));
+    return element(by.xpath('//li[@class="x caps" and contains(text(),"assets")]'));
```
Comment (pspani) [inline core/jazz_ui/e2e/page-objects/jazzservices.po.ts]: good change. Will save us having to fix the tests even if tabs are added to removed in the future

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts (hunk 2 above)
Comment (mayajohns) [reply core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: We can reduce that,for seeing the Logs I just kept 4 seconds.

**Pull request merged**